### ISSUE TITLE
Init is not working in 3.0.0.0/1

### ISSUE THREAD

**Issue by veccie** [GitTools/GitVersion#545]
ConsoleAdapter from #532 doesn't seem to be fully implemented yet. Running GitVersion init gives the following error:
```
An unexpected error occurred:
System.NotImplementedException: The method or operation is not implemented.
   at GitVersion.ConsoleAdapter.WriteLine(String msg) in c:\projects\gitversion\src\GitVersionCore\Configuration\ConsoleAdapter.cs:line 9
   at GitVersion.Configuration.Init.Wizard.ConfigInitWizard.Run(Config config, String workingDirectory) in c:\projects\gitversion\src\GitVersionCore\Configuration\Init\Wizard\ConfigInitWizard.cs:line 19
   at GitVersion.ConfigurationProvider.Init(String workingDirectory, IFileSystem fileSystem, IConsole console) in c:\projects\gitversion\src\GitVersionCore\Configuration\ConfigurationProvider.cs:line 117
   at GitVersion.Program.VerifyArgumentsAndRun() in c:\projects\gitversion\src\GitVersionExe\Program.cs:line 74
```

**Comment by JakeGinnivan** [comment]
Doh. Thanks for the report. Will get 3.0.2 out later this morning

**Comment by JakeGinnivan** [comment]
Fixed by #549
